scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -130,8 +133,15 @@ if [ "$silent" == "true" ]; then
 else
     kubectl ws "${kubectl_flags[@]}" "$imw"
 fi
+spaceID=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}")
 
-mbwsname=$(kubectl "${kubectl_flags[@]}" get synctargets.edge.kubestellar.io "$stname" -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}")
+if [ "$central" == "true" ]; then
+    echo "looking for synctargets in the center"
+    kubectl ws "${kubectl_flags[@]}" "$espw"
+fi
+prefixed_stname="$(kubectl get synctargets.edge --no-headers=true -o custom-columns=":metadata.name" | grep $stname)"
+stUID=$(kubectl get synctargets.edge $prefixed_stname -o jsonpath="{.metadata.uid}")
```

Comment:
`"${kubectl_flags[@]}"` seems to have been forgotten in a lot of places.

Reply:
Because a kubectl plugin should accept kubectl command line flags.

Reply:
May I have an example?

Reply:
explicit `--kubeconfig` on the command line. Or `--context`.

Reply:
I see. Thanks.

Reply:
I checked through all scripts whose names begin with `kubectl-` in this PR, looks like `"${kubectl_flags[@]}"` are added now.

Reply:
yes